rust/src/domains/polars/csv/mod.rs

```diff
@@ -111,18 +120,66 @@ impl CsvDomain {
             .with_end_of_line_char(self.eol_char as u8)
             .with_null_values(self.null_values.clone())
     }
+
+    pub fn new_writer(&self, path: PathBuf) -> CsvWriter<File> {
+        let file = File::create(path).unwrap();
```

Comment:
Lets avoid this unwrap- if a user passes a path with dirs that don't exist, it would manifest as the entire python/R process terminating.
https://doc.rust-lang.org/std/fs/struct.File.html#method.create